libfabric.spec.in

```diff
@@ -1,6 +1,6 @@
 Name: libfabric
 Version: @VERSION@
-Release: 1%{?dist}
+Release: 1.rc1%{?dist}
```

Comment:
Also now might be a good time to hard code the configure flags which choose the providers and corresponding rpm buildrequires